cogs5e/models/sheet/player.py

```diff
@@ -94,9 +94,13 @@ class CustomCounter:
                  'hp': "Gaining HP"}
 
     def __init__(self, character, name, value, minv=None, maxv=None, reset=None, display_type=None, live_id=None,
-                 reset_to=None, reset_by=None):
+                 title=None, desc=None, reset_to=None, reset_by=None):
```

Comment:
```suggestion
                 reset_to=None, reset_by=None, title=None, desc=None):
```

I don't think `__init__` is ever called with only positional args, but this prevents breaking it if it is.